net.py

```diff
@@ -1,172 +0,0 @@
-import numpy as np
-
-class Net:
-    def __init__(self, layers):
-        self.W = []
-        self.dW = []
-        self.b = []
-        self.db = []
-        self.A_saved = []
-        self.Z_saved = []
-        self.dA = []
```

Comment:
Consider using of subclasses like point, vector etc. It's more readable then one thousand of array